### PR TITLE
Display info snackbar on study rename or move

### PR DESCRIPTION
**Pull request by Meklo**

### CODE REVIEW

File: src/utils/compute-title.js [hunk 1]
```diff
@@ -8,6 +8,16 @@
 const MAX_TITLE_LENGTH = 106;
 const SEPARATOR = ' | ';
 
+export const computeFullPath = (parents) => {
+    let testedPath = '';
+    let path = '';
```
Comment (FranckLecuyer) [inline src/utils/compute-title.js:13]: 'path' variable seems not necessary

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]
Tests:OK

File: src/utils/compute-title.js (hunk 1 above)
Comment (Meklo) [reply src/utils/compute-title.js:12]: One of the variables is indeed useless, I removed testedPath and kept path for semantics reasons

**Review by FranckLecuyer — APPROVED** [review]

File: src/components/study-container.js [hunk 2]
```diff
@@ -356,22 +358,45 @@ export function StudyContainer({ view, onChangeTab }) {
...
                 document.title = computePageTitle(
                     initialTitle,
-                    study?.elementName,
+                    studyName,
                     parents
                 );
+
+                studyMetadata.path = path;
+                studyMetadata.name = studyName;
+                setStudyMetadata(studyMetadata);
```
Comment (archendel) [inline src/components/study-container.js:393]: is it safe to call setStudyMetadata here whereas the useCallBack has studyMetadata in dependency array ?

**Review by archendel — APPROVED** [review]
One small remark

**Comment by Meklo** [comment]
> One test which does not work, but maybe it's out of this PR : If you rename one of the study parents, it does not update anything on gridstudy
> 
> Also, one remark below

It will indeed be the subject of a future PR

File: src/components/study-container.js (hunk 2 above)
Comment (Meklo) [reply src/components/study-container.js]: Upon research it wasn't best practice compliant so I've overhauled the process through a custom hook

**Pull request merged**